doc/developer-guides/hld/hv-dev-passthrough.rst

```diff
@@ -102,12 +125,32 @@ following figures:
 
    ptdev de-assignment control flow
 
+VT-d Interrupt-remapping
+************************
+
+The VT-d interrupt-remapping architecture enables system software to
+control and censor external interrupt requests generated by all sources
+including those from interrupt controllers (I/OxAPICs), MSI/MSI-X capable
+devices including endpoints, root-ports and Root-Complex integrated
+end-points.
+ACRN forces to enabled VT-d interrupt-remapping feature for security reasons.
```

Comment:
@dbkinder , shouldn't this be rephrased a bit? Maybe something like "ACRN enforces that the VT-d interrupt-remapping feature be enabled for security reasons"